specification/dartLangSpec.tex

```diff
@@ -2786,8 +2790,8 @@ \subsection{Setters}
 }
 
 \LMHash{}%
-It is a static warning if a setter declares a return type other than \VOID{}.
-It is a static warning if a class has
+It is a compile-time error if a setter declares a return type other than \VOID.
```

Comment:
The `[]=` text says "explicitly declared" and this just says "declared".

Consider making the phrasing consistent, so readers won't think there is a meaningful difference.

Reply:
Adjusted the text about `[]=` --- we don't have any other occurrence of 'explicitly declare{s,d}' in the language specification outside commentary.